data/banners.yaml

```
@@ -0,0 +1,11 @@
+- title: 'OTel Community Day'
+  message:
+    '[**OTel Community Day** on June
+    25th](https://sessionize.com/OTel-Community-Day/)!'
+  endDate: 2024-06-25
+
+- title: 'OTel and Prometheus Survey'
+  message:
+    'Help improve OTel and Prometheus interoperability: [complete our
+    survey](https://forms.gle/bZAG9f7udoJsjZUG9) by May 31'
+  endDate: 2024-05-31
```

```suggestion
  endDate: 2024-05-31

- title: Roadmap to v1 for the OpenTelemetry Collector
  message: '[Learn about our roadmap to v1 for the OpenTelemetry Collector](/blog/2024/collector-roadmap/)'
  endDate: 2024-05-21
```

Can you add that for experimentation? If the code works as expecrted it should drop the community day for now